src/test/java/software/amazon/smithy/lsp/document/DocumentParserTest.java

```diff
@@ -67,6 +74,8 @@ public void jumpsToLines() {
         assertEquals(1, parser.column());
     }
 
+
+
```

Comment:
Formatting (probably need to set up some linting rules for this)

Reply:
Yes. Unfortunately we dont have a consistent formatter. Let me see if we can add one as part of a separate PR?